mifospay/src/main/java/org/mifos/mobilewallet/mifospay/history/HistoryContract.java

```diff
@@ -31,6 +31,8 @@
         void showTransactionDetailDialog(int transactionIndex, String accountNumber);
 
         void showHistoryFetchingProgress();
+
+        void scrollDownBottom();
```

Comment:
The name of the function should reflect what you are doing in the function. In this case scrollDownBottom gives the impression that the function will be called once you need to scroll to bottom, but that's not the case in the actual implementation. 
Instead a more explanatory function name, like say setupScrollDownBottomButton will lead to better readability of code.

Reply:
What say of the name scrollDownBottomButton()?

Reply:
@rahul-jha98 updated the changes, if you want to have a look :)

Reply:
The other changes looks fine to me. But are you sure you have renamed the function. I see its still named the same as before. Please look into this.

Reply:
@rahul-jha98 Sorry sir, seems like I have missed it. I have changed it now :)